### PR TITLE
styling

### PR DESCRIPTION
**Pull request by jw122**
Before: 
image

After:
image

### CODE REVIEW

File: src/web3/wallet.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Web3 from "web3";
 // as other block chain information
 export const initializeTorusConnection: () => Promise<Torus> = async (): Promise<Torus> => {
   const torus = new Torus({
-    buttonPosition: "top-left", // default: bottom-left
+    // buttonPosition: "top-left", // default: bottom-left
```
Comment (adotcruz) [inline src/web3/wallet.ts]: We can delete this object.

File: src/components/account/UserHoldings.tsx [hunk 2]
```diff
@@ -9,14 +9,35 @@ declare interface UserHoldingsProps {
...
+          <img src={tokenIcons[token.symbol]} alt="" width="15px" />{" "}
+          {token.label}
+        </h5>
+        <div>
+          {token.balance.toFixed(4)} (${token.balanceUSD.toFixed(4)} USD)
+        </div>
+        <div>
+          {token.apy ? (
+            <p>
+              {" "}
```
Comment (adotcruz) [inline src/components/account/UserHoldings.tsx]: what does this do?

**Review by adotcruz — APPROVED** [review]

File: src/components/account/UserHoldings.tsx (hunk 2 above)
Comment (jw122) [reply src/components/account/UserHoldings.tsx]: not sure how these got added. Deleted now

File: src/web3/wallet.ts (hunk 1 above)
Comment (jw122) [reply src/web3/wallet.ts]: done

**Pull request merged**